tox.ini

```diff
@@ -19,13 +19,15 @@ deps =
     sybil >= 1.3.0  # https://github.com/cjw296/sybil/issues/20#issuecomment-605433422
     testfixtures
     pywin32; sys_platform == "win32"
+    cffi >= 1.17.0rc1; python_version >= '3.13'
 
 [testenv]
 deps =
     {[test-requirements]deps}
 
     # mitmproxy does not support PyPy
-    mitmproxy; implementation_name != 'pypy'
+    # mitmproxy requires zstandard which is not yet available on 3.13
+    mitmproxy; implementation_name != 'pypy' and python_version < '3.13'
```

Comment:
To remove after a full cffi release is made (planned in July) and zstandard wheels are built with it.